test/test_unit_import_workflow.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
Did you think about doing an integration test that does CWL → Python → CWL and comparing the input and output CWL?

Reply:
I think it would be a good test to improve this translation mechanism. For this change I know it won't produce identical CWL because it's opinionated on the dict / array output of inputs / outputs / steps.

It would be good to functionally validate the CWL tbh.